### PR TITLE
Make TimeMon's border color configurable

### PR DESCRIPTION
**Pull request by onflapp**

### CODE REVIEW

**Comment by onflapp** [comment]
small change but it can make TimeMon look much better (IMHO)
screen-shot7480

File: Applications/TimeMon/Percentages.m [hunk 1]
```diff
@@ -261,6 +266,7 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification
            @"UserColor":@"0.200 0.467 0.800 1.000",     // A darker blue-green
            @"SystemColor":@"0.000 0.000 1.000 1.000",   // Blue
            @"IOWaitColor":@"1.000 0.800 0.900 1.000",   // Light purple
+           @"BorderColor":@"0.333",                     // Default border color
```
Comment (trunkmaster) [inline Applications/TimeMon/Percentages.m:269]: Please leave default border color black.

**Review by trunkmaster — COMMENTED** [review]
I think we could start a contest for best default color TimeMon design. ;) Let's keep default colors unchanged for now.

File: Applications/TimeMon/Percentages.m (hunk 1 above)
Comment (onflapp) [reply Applications/TimeMon/Percentages.m]: Sure, makes sense.

**Pull request merged**